app/apis/v1/record_sets_api.rb

```diff
@@ -0,0 +1,76 @@
+module V1
+
+    class RecordSetsApi < Grape::API
+  
+      resource :record_sets do
+  
+        params do
+          requires :functions, type: String # 選擇要什麼 Function: Sets / Times
+          requires :user_id, type: String
+          requires :contents, type: String
+          requires :sets, type: Integer # 組數
+          requires :reps, type: Integer # 次數
+          requires :weight, type: Float # 重量
+          requires :record_sets_id, type: Interger # 應該要在新增時生成
```

Comment:
`          requires :record_sets_id, type: Interger # 應該要在新增時生成`
It will generate when the data saves to DB, so users do not have to post id to API. right?